src/prisma/migrations/20230927004337_transactions_cancelled_at_added/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- AlterTable
```

Comment:
Any reason this migration is here with these smart wallet fields at the beginning? Should be included in previous ones right - maybe local DB out of sync?

Reply:
I will remove this and create a new migration with only `cancelledAt`.

This alter for added as I synced from main sometime back which has this change & my local didn't.

Reply:
Now that I think of this, if I remove this then I will not have the type change `INTEGER` to `TEXT`. 

migrations -> `20230922090743_user_ops` -> Adds the column `callGasLimit` as `INTEGER`
migrations -> `20230927004337_transactions_cancelled_at_added` -> updates few columns `INTEGER` to `TEXT` & adds `cancelledAt` field

@adam-maj